server.api/src/main/java/org/kaazing/gateway/server/impl/VersionUtils.java

```diff
@@ -238,4 +203,35 @@ private static boolean getProductInfoFromJar(String pathEntry, Map<String, Attri
         }
         return result;
     }
+
+    private static class GatewayProductVersionType {
+        int digits;
```

Comment:
I believe it is better if you use title, version, edition, dependencies as membersi nstead of just digits.